scripts/Muon/GUI/Common/contexts/muon_gui_context.py

```diff
@@ -53,9 +53,4 @@ def add_non_calc_subscriber(self, observer):
         self.gui_variable_non_calulation_notifier.add_subscriber(observer)
 
     def remove_workspace_by_name(self, workspace_name):
-        try:
-            if self['DeadTimeTable'].name() == workspace_name:
-                self.pop('DeadTimeTable')
-                self['DeadTimeSource'] = 'FromFile'
-        except KeyError:
-            pass
+        pass
```

Comment:
If we are now just passing when calling this function do we really need to keep it? The code base is already quite confusing and I worry that functions which just pass could lead to more confusion. If it is too much hassle to remove now we can leave this for maintenance at a later date as this PR is holding up other work